setup.cfg

```diff
@@ -27,6 +26,7 @@ classifiers =
 
 [options]
 include_package_data = True
+python_requires = >3.7
```

Comment:
Probably should be `>=3.7`, actually. The thought here is that we haven't actually dropped Python 3.7 support quite yet, although we're not testing on it. Since there's an opcode change between 3.7 and 3.8, I want to stretch the window just a little longer for anyone who started on 3.7. (Even after opcode changes, you can reload old data with safemode, but that's a hassle.) This is especially important because we had sometimes been slow to officially support new versions of Python.

I'm hoping to get to a 1.6 release in the next few weeks, and with that off my shoulders, more frequent releases should be easier. I'd like this to be `>=3.9` by the end of the year, and then to actually follow the NEP29 drop schedule.